src/test/e2e/helpers/test-helper.ts

```diff
@@ -45,15 +55,29 @@ function httpApis(
 
     return {
         createFeature: (
-            name: string,
+            feature: string | CreateFeatureSchema,
             project: string = DEFAULT_PROJECT,
             expectedResponseCode: number = 201,
         ) => {
+            let body = feature;
+            if (typeof feature === 'string') {
+                body = {
+                    name: feature,
+                };
+            }
```

Comment:
I'm not a big fan of having this logic here, but to make it simpler in testing I don't think it hurts as much as if I'm doing this in production code.

Reply:
Also a bit unsure about this one, my mind is telling me maybe we should have both `createFeature` and e.g. `createFeatureByName` methods with strict typing, where the second one does this for us before sending it in to `createFeature`. 

But it's like you said, maybe it doesn't hurt as much as we're thinking.

Reply:
I'm more inclined to remove the create feature by name and use the correct schema. But, it's more like a shortcut. In Java I'd use method overloading, but I think this is the best way of accomplishing this in Typescript (or alternatively having 2 functions with different names)

Reply:
Yeah exactly. I like the idea of having this helper for easier use that allows creating features simply by passing in the name. Whether that's through a different method that refactors things to be stricter typed or this single method that handles both cases, I'll leave it up to you.

Reply:
I think this is fine for now, we can continue to refactor later on